src/ReactAI.ts

```diff
@@ -1,32 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { PropertiesPluginIdentifier } from "@microsoft/applicationinsights-common";
 import { IPlugin } from "@microsoft/applicationinsights-core-js";
-import {
-  IAppInsightsCore,
-  IApplicationInsights,
-  IConfig,
-  IConfiguration,
-  IPageViewTelemetry,
-  ITelemetryItem,
-  ITelemetryPlugin
-} from "@microsoft/applicationinsights-web";
+import { ApplicationInsights as AppInsightsPlugin, IAppInsightsCore, IConfig, IConfiguration, IPageViewTelemetry, ITelemetryItem, ITelemetryPlugin, PropertiesPlugin } from "@microsoft/applicationinsights-web";
 import { Action, History, Location } from "history";
 import IReactAISettings from "./IReactAISettings";
 
+
 /**
  * Module to include Microsoft Application Insights in React applications.
  *
  * @export
- * @class ReactAI
+ * @class ReactAI  
  */
 class ReactAI implements ITelemetryPlugin {
   public extensionId: string = "ApplicationInsightsReactUsage";
   public ApplicationInsightsAnalyticsIdentifier: string = "ApplicationInsightsAnalytics";
   public processTelemetry: (env: ITelemetryItem) => void;
   public identifier = this.extensionId;
   public priority: number = 190;
-  public appInsights!: IApplicationInsights;
+  public appInsights!: AppInsightsPlugin;
```

Comment:
You want this to be public for the consumers to be able to access?

Reply:
There is nothing private about the internal IAppInsights. the public layer is a combination of multiple of such interfaces.